src/components/layout/nav/components/logo-section.tsx

```diff
@@ -54,10 +61,16 @@ const LogoSection = ({ is_ppc_redirect, base }: LogoSectionProps) => {
                     data={data['deriv']}
                     alt="deriv logo"
                     max_width="16.4rem"
-                    width="96px"
+                    width="100%"
                     height="auto"
                 />
             </LogoLink>
+            {!hide_get_trading && (
```

Comment:
```suggestion
            {!hide_get_trading ? (...):(null)
```

Reply:
https://medium.com/geekculture/stop-using-for-conditional-rendering-in-react-a0f7b96200f8